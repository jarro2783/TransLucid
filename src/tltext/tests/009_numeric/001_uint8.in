%%
uint8"0";;
uint8"1";;
uint8"1" + uint8"5";;
uint8"127";;
uint8"255";;
uint8"0" + uint8"255";;

uint8"10" - uint8"4";;

uint8"23" / uint8"5";;

uint8"7" % uint8"4";;

uint8"15" < uint8"16";;
uint8"15" < uint8"15";;
uint8"15" > uint8"16";;
uint8"15" > uint8"15";;
uint8"15" >= uint8"16";;
uint8"15" >= uint8"15";;
uint8"15" <= uint8"16";;
uint8"15" <= uint8"15";;
uint8"15" == uint8"15";;
uint8"15" == uint8"16";;
