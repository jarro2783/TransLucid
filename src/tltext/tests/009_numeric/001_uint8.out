uint8"0"
uint8"1"
uint8"12"
uint8"255"
uint8"0"
uint8"15"
uint8"15"
uint8"13"
uint8"0"
uint8"10"
uint8"12"
uint8"6"
uint8"0"
uint8"5"
uint8"4"
uint8"3"
true
false
false
true
true
false
false
false
true
false
true
true
false
true
false
true
false
true
8
false
